scripts/Connecting.gd

```diff
@@ -0,0 +1,19 @@
+extends ColorRect
```

Comment:
You may want to think about coupling these scripts files into logical groups and create sub-directories for them, otherwise /scripts is going to become very hard to navigate.

Reply:
Made the structure changes. I'm pretty bad at organising files into sensible directories, so *please* tell me if it's a poor approach. 

Also, somewhere in there is a reference to the `Connecting.gd` script that got broken. Everything seems to work just fine, so I dunno what it's doing, but we do know exactly where it is:

```
 modules/gdscript/gdscript.cpp:829 - Condition "err" is true. Returned: err
 Cannot load source code from file 'res://scripts/Connecting.gd'.
 Failed loading resource: res://scripts/Connecting.gd. Make sure resources have been imported by opening the project in the editor at least once.
```

It's probably fine. Just annoying.